content/attr/contents.lr

```diff
@@ -41,3 +41,5 @@ body2:
 </div>
 
 <br>
+
+Raw <a href="https://github.com/terminalguide/terminalguide/blob/master/content/attr/example.txt" target="_blank">example.txt</a> to test color in a termial.
```

Comment:
I think it would be better to link to the raw file on github instead. Also i think this would be better phrased in a way to say this file is the "source" of the sample images.